.github/workflows/tests.yml

```
@@ -160,7 +160,8 @@ jobs:
   centos-8:
     name: CentOS 8
     runs-on: ubuntu-18.04
-    container: centos:8
+    # This centOS 8 image
+    container: quay.io/centos/centos:stream8
```

Whoops, forgot to finish my own comment.

```suggestion
    # Stream8 has a 2024 EOL, while CentOS 8 has a 2021 EOL
    # See: https://blog.centos.org/2020/12/future-is-centos-stream/
    container: quay.io/centos/centos:stream8
```

---

Either way, this change did indeed fix the issue: https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/5012784329?check_suite_focus=true